composer.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
I spent a while getting [pie](https://github.com/php/pie) running locally just to find out that it doesn't support a `validate` command to check this JSON file. Well, at least I'll have it ready for future use :shrug:

Reply:
I'll use the opportunity to summon @asgrim, as they'd be in a better place to give you an idea of if and when to expect a `pie validate` command.

Reply:
You can actually just use `composer validate`; the schema for the `php-ext` bit is actually built in there.